simulariumio/constants.py

```diff
@@ -24,7 +24,7 @@ class V1_SPATIAL_BUFFER_STRUCT:
     R_INDEX: int = 9  # radius
     NSP_INDEX: int = 10  # num subpoints
     SP_INDEX: int = 11  # subpoints
-    VALUES_PER_AGENT: int = 12
+    VALUES_PER_AGENT: int = 11
```

Comment:
why is this changing?

Reply:
This is the only scary one to me.  It really looks like the number of values per agent should be 12.

Reply:
12 included the first subpoint value. If the number of subpoints is 0, there will be no subpoint values. In a lot of the code I updated, it had been implemented using `VALUES_PER_AGENT - 1`